app/controllers/api/admin/patients_controller.rb

```diff
@@ -22,9 +22,21 @@ def count
 
       api :POST, "/admin/patients", "Upload a zip file of patients."
       param :file, nil, :desc => 'The zip file of patients to upload.', :required => true
+      param :practice_id, nil, :desc => "ID for the patient's Practice", :required => false
+      param :practice_name, nil, :desc => "Name for the patient's Practice", :required => false
+      
       def create
         file = params[:file]
-
+        
+        if params[:practice_id]
+          practice = params[:practice_id]
+        elsif params[:practice_name]
+          ext = Practice.where(name: params[:practice_name]).first
+          practice =  ext ? ext._id.to_s : nil
```

Comment:
Whitespace is off.

Consider replacing with practice = ext.try(:_id).to_s. We attempt to avoid the ternary operator in the codebase.